examples/02_classification.ipynb

```diff
@@ -18,19 +25,20 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# 2. Time Series Classification, Regression, Clustering & more <a name=\"top-2\"></a>\n",
     "\n",
-    "deal with *collections of time series* = \"panel data\"\n",
+    "Here our primary aim is to deal with *collections of time series* = \"panel data\"\n",
```

Comment:
I would prefer this shorter, telegram style - this can be read directly with the title

Reply:
Reverted back to "Deal with collections of time series"